packages/astro/src/@types/astro.ts

```diff
@@ -2181,6 +2181,258 @@ export interface AstroUserConfig {
+		 *     pubDate: z.coerce.date(),
+		 *     updatedDate: z.coerce.date().optional(),
+		 *   }),
+		 * });
+		 *
+		 * export const collections = { blog };
+		 * ```
+		 *
+		 * You can load multiple entries from a single JSON file using the `file()` loader. In this case the data must either be an array of objects, which each contain an `id` property, or an object where each key is the ID.
+		 *
+		 * **Array syntax:**
+		 *
+		 * ```json
+		 * [
+		 *   {
+		 *     "id": "labrador-retriever",
+		 *     "breed": "Labrador Retriever",
+		 *     "size": "Large",
+		 *     "origin": "Canada",
+		 *     "lifespan": "10-12 years",
+		 *     "temperament": [
+		 *       "Friendly",
+		 *       "Active",
+		 *       "Outgoing"
+		 *     ]
+		 *   },
+		 *   {
+		 *     "id": "german-shepherd",
+		 *     "breed": "German Shepherd",
+		 *     "size": "Large",
+		 *     "origin": "Germany",
+		 *     "lifespan": "9-13 years",
+		 *     "temperament": [
+		 *       "Loyal",
+		 *       "Intelligent",
+		 *       "Confident"
+		 *     ]
+		 *   }
+		 * ]
+		 * ```
+		 *
+		 * **Object syntax:**
+		 *
+		 * ```json
+		 * {
+		 *   "labrador-retriever": {
+		 *     "breed": "Labrador Retriever",
+		 *     "size": "Large",
+		 *     "origin": "Canada",
+		 *     "lifespan": "10-12 years",
+		 *     "temperament": [
+		 *       "Friendly",
+		 *       "Active",
+		 *       "Outgoing"
+		 *     ]
+		 *   },
+		 *   "german-shepherd": {
+		 *     "breed": "German Shepherd",
+		 *     "size": "Large",
+		 *     "origin": "Germany",
+		 *     "lifespan": "9-13 years",
+		 *     "temperament": [
+		 *       "Loyal",
+		 *       "Intelligent",
+		 *       "Confident"
+		 *     ]
+		 *   }
+		 * }
+		 * ```
+		 *
+		 * The collection is then defined using the `file()` loader:
+		 *
+		 * ```ts
+		 * import { defineCollection, z } from 'astro:content';
+		 * import { file } from 'astro/loaders';
+		 *
+		 * const dogs = defineCollection({
+		 * 	loader: file('src/data/dogs.json'),
+		 * 	schema: z.object({
+		 * 		id: z.string(),
+		 *     breed: z.string(),
+		 * 		size: z.string(),
+		 * 		origin: z.string(),
+		 * 		lifespan: z.string(),
+		 * 		temperament: z.array(z.string()),
+		 * 	}),
+		 * });
+		 *
+		 * export const collections = { dogs };
+		 * ```
+		 *
+		 * The collection can be queried in the same way as content collections:
+		 *
+		 * ```ts
+		 * import { getCollection, getEntry } from 'astro:content';
+		 *
+		 * // Get all entries from a collection.
+		 * // Requires the name of the collection as an argument.
+		 * const allBlogPosts = await getCollection('blog');
+		 *
+		 * // Get a single entry from a collection.
+		 * // Requires the name of the collection and ID
+		 * const labradorData = await getEntry('dogs', 'labrador-retriever');
+		 * ```
+		 *
+		 * #### Rendering content
+		 *
+		 * Entries generated from markdown or MDX can be rendered directly to a page using the `render()` function.
+		 *
+		 * > [!WARNING]
```

Comment:
Do these work in our docs?

Reply:
I don't think so, because it's GitHub only syntax

Reply:
https://starlight.astro.build/guides/authoring-content/#asides